app/assets/v2/js/pages/profile.js

```diff
@@ -0,0 +1,163 @@
+$(document).ready(function() {
+  $('[data-toggle=popover]').popover({
+    html: true,
+    trigger: 'focus',
+    template: '<div class="popover wallet-popover" role="tooltip"><div class="arrow"></div><h3 class="popover-header"></h3><div class="popover-body"></div></div>',
+    content: function() {
+      var content = $(this).attr('data-popover-content');
+
+      return $(content).children('.popover-body').html();
+    },
+    title: function() {
+      var title = $(this).attr('data-popover-content');
+
+      return $(title).children('.popover-heading').html();
+    }
+  });
+
+  if (!document.hasPreferredPayoutAddress) {
+    $('[data-toggle=popover]').popover('show');
+  }
+
+  $('body').on('submit', '#form-wallets', function(e) {
+    e.preventDefault();
+
+    var data = $('#form-wallets').serializeArray();
+
+    data.push({'page': 2});
```

Comment:
is page=2 hardcoded the correct way to go here?

Reply:
I actually just lint'ed that code and moved it to the dedicated js file.
That's from the Kudos introduction.